Tooling-Landscape/Unanimous-Understanding/OSS_Tooling_Landscape_UMLComp.plantuml

```diff
@@ -59,6 +59,13 @@ node "Company Internal" <<Company Network>> {
 
     component [Security Vulnerability Assessment] as securityassessment <<Internal Service>>
 
+    component [Forensic Code Analyzer] as forensiccodeanalyzer <<Internal Service>>
```

Comment:
Two comments:
1. I prefer to have the relationships after the component description like the hidden links below. Perhaps you create a section before the hidden stuff for the visual relations.
2. There are two additional use relations
`componentanalysis ..> licensescanner : use`
`componentanalysis ..> forensiccodeanalyzer : use`